<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginSettingDialog</class>
 <widget class="QWidget" name="PluginSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbEnablePlugin">
        <property name="text">
         <string>EnablePlugin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEnablePluginRoot">
        <property name="text">
         <string>EnablePluginRoot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/com.wingsummer.winghex/images/plugin.png</normaloff>:/com.wingsummer.winghex/images/plugin.png</iconset>
      </attribute>
      <attribute name="title">
       <string>PluginInfo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Plugins:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="plglist"/>
           </item>
          </layout>
         </widget>
         <widget class="QTextBrowser" name="txtc">
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget">
      <attribute name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/com.wingsummer.winghex/images/devext.png</normaloff>:/com.wingsummer.winghex/images/devext.png</iconset>
      </attribute>
      <attribute name="title">
       <string>DevExtInfo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_2">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>DevExt:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="devlist"/>
           </item>
          </layout>
         </widget>
         <widget class="QTextBrowser" name="txtd">
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/com.wingsummer.winghex/images/monitor.png</normaloff>:/com.wingsummer.winghex/images/monitor.png</iconset>
      </attribute>
      <attribute name="title">
       <string>APIMonitor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTextBrowser" name="txtm">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
